src/Attributes.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-view for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-view/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-view/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\View;
+
+use ArrayObject;
+use Laminas\Escaper\Escaper;
+use Traversable;
+
+/**
+ * Class for storing and processing HTML tag attributes.
+ */
+class Attributes extends ArrayObject
+{
+    /**
+     * HTML escaper
+     *
+     * @var Escaper
+     */
+    protected $htmlEscaper;
+
+    /**
+     * HTML attribute escaper
+     *
+     * @var Escaper
+     */
+    protected $htmlAttrEscaper;
+
+    /**
+     * Constructor.
+     *
+     * @param Escaper $htmlEscaper HTML escaper
+     * @param Escaper $htmlAttrEscaper Attribute escaper
+     * @param array|Traversable $attribs Attributes
+     */
+    public function __construct($htmlEscaper, $htmlAttrEscaper, $attribs = [])
```

Comment:
```suggestion
     * @param Escaper $htmlEscaper General HTML escaper
     * @param Escaper $htmlAttrEscaper Escaper for use with HTML attributes
     * @param iterable $attribs Attributes to manage
     */
    public function __construct(Escaper $htmlEscaper, Escaper $htmlAttrEscaper, iterable $attribs = [])
```

The 2.12.x sets the minimum supported PHP version at 7.3, so new features and classes, such as this one, can use the full range of features from that version.